fdbcli/tests/fdbcli_tests.py

```diff
@@ -1067,6 +1067,11 @@ def tenant_configure(logger):
     output = run_fdbcli_command("tenant configure tenant tenant_group=tenant_group1")
     assert output == "The configuration for tenant `tenant' has been updated"
 
+    output = run_fdbcli_command(
```

Comment:
Maybe also test the case of feeding the following
```
tenant configure tenant tenant_group=tenant_group1 ingore_capacty
```
to make sure typo is properly reported.
I checked the code, and it seems so, but would be nice to use a test to guard against it.

Reply:
This uses the same path as another test, which is to report an error if you have a configuration parameter that doesn't specify `=`. If the name of this doesn't match the desired token, it will be treated as a normal configuration parameter.

Reply:
I added a second slightly different version of the invalid configuration token test